inductiva/molecules/scenarios/utils.py

```diff
@@ -1,11 +1,19 @@
 """Tools to analyze molecular dynamics simulations."""
-import MDAnalysis as mda
-from MDAnalysis import transformations
-from MDAnalysis.analysis import align
+try:
+    import MDAnalysis as mda
+    from MDAnalysis import transformations
+    from MDAnalysis.analysis import align
+except ImportError:
+    mda = None
```

Comment:
globals?

Reply:
these were the imports that were already there. I just wrapped them in try except to catch the ImportError